src/map/lua/lua_baseentity.cpp

```diff
@@ -16660,6 +16673,36 @@ bool CLuaBaseEntity::hasTPMoves()
     return !MobSkills.empty();
 }
 
+/************************************************************************
+ *  Function: drawIn()
+ *  Purpose : Draws in the target, or current target if not specified
+ *  Example : mob:drawIn()     mob:drawIn(player)
+ *  Notes   : Draws in a player even if within the draw-in leash
+ ************************************************************************/
+void CLuaBaseEntity::drawIn(sol::variadic_args va)
+{
+    auto mobObj = dynamic_cast<CMobEntity*>(m_PBaseEntity);
+
+    if (va.size() == 0)
+    {
+        auto defaultTarget = mobObj->GetBattleTarget();
+
+        battleutils::DrawIn(defaultTarget, mobObj, mobObj->GetMeleeRange() - 0.2f);
```

Comment:
What happens if `defaultTarget` is null?

Reply:
Good catch.  Added a check for null and just returns silently
```cpp
        auto defaultTarget = mobObj->GetBattleTarget();

        if (defaultTarget == nullptr)
        {
            return;
        }
        battleutils::DrawIn(defaultTarget, mobObj, mobObj->GetMeleeRange() - 0.2f);
        return;
```